src/promisifyDrmAgent.ts

```diff
@@ -0,0 +1,61 @@
+/// <reference types="@procot/webostv/webOSTV-dev/drmAgent" />
+
+import { RequestCallback } from './types';
+import { promisifyRequest } from './promisifyRequest';
+
+/**
+ * Wraps a original asyncronous methods in instance `DRMAgent` object or nested object in Promise
+ *
+ * http://webostv.developer.lge.com/api/webostvjs/webosdev/drmagent/
+ *
+ * @param drmAgent instance of `DRMAgent`
+ * @returns promised `DRMAgent` instance
+ */
+export function promisifyDrmAgent(drmAgent: DRMAgent): DRMAgentPromised {
+  const drmAgentPromised: DRMAgentPromised = Object.assign(drmAgent);
+
+  Object.defineProperties(drmAgentPromised, {
```

Comment:
Why `Object.defineProperties` not `Object.assign`?

Reply:
The all fields of webOS and webOSDev objects are getters that returns function. So should use Object.defineProperties to rewrite fields.

Reply:
ok!